src/components/PurchaseOrder/POForm.js

```diff
@@ -310,9 +310,9 @@ class POForm extends Component {
                         {isOngoing(formValues.orderType) && (
                           <Accordion
                             id="renewals"
-                            label={<FormattedMessage id="ui-orders.paneBlock.renewals" />}
+                            label={<FormattedMessage id="ui-orders.paneBlock.ongoingInfo" />}
                           >
-                            <RenewalsForm order={initialValues} />
+                            <OngoingInfoForm order={initialValues} formValues={formValues.ongoing} />
```

Comment:
`formValues={formValues.ongoing}` looks confusing to me. It's not a separate form, it's a sub-form. Maybe `ongoingFormValues` or something is better